build/profiles/freenas/ports-system.pyd

```diff
@@ -340,6 +337,7 @@ ports += {
 }
 
 ports += "sysutils/lsof"
+ports += "net/avahi-app"
```

Comment:
Can we add this to freenas/py-middlewared instead?

This ports-system file is legacy and we should eventually empty this completely.